src/MSBuild.SDK.SystemWeb.CommonFiles/Sdk/MSBuild.SDK.SystemWeb.Common.DefaultPackages.props

```diff
@@ -2,5 +2,8 @@
   <PropertyGroup Condition="'$(ExcludeSDKDefaultPackages)'=='false'">
     <MicrosoftNetCompilersToolset_Version Condition="'$(MicrosoftNetCompilersToolset_Version)'==''">4.5.0</MicrosoftNetCompilersToolset_Version>
     <MicrosoftCodeDomProvidersDotNetCompilerPlatform_Version Condition="'$(MicrosoftCodeDomProvidersDotNetCompilerPlatform_Version)'==''">3.6.0</MicrosoftCodeDomProvidersDotNetCompilerPlatform_Version>
+    <MicrosoftAspNetMvc_Version Condition="'$(MicrosoftAspNetMvc_Version)'==''">5.2.9</MicrosoftAspNetMvc_Version>
```

Comment:
I see you have moved these from the RazorLibrary specific DefaultPackages files.
As these are not used in the main SDK (currently), I'm not sure that I like having them defined without purpose.
My idea was that all common packages/versions would be in Common, and any specific to either the main SDK or RazorLibrary SDK would be in the specific one.
Is there a reason for this move that I am missing?

Reply:
No special purpose at the moment. I'll put them back

Reply:
I think you've copied them back, but not removed them here.

Reply:
Done